src/commands/scan.ts

```diff
@@ -0,0 +1,149 @@
+import {Command, flags} from '@oclif/command';
+import {Scanner, ScanResult} from '../scanner';
+import cli from 'cli-ux';
+import {RuleContext} from '../rules';
+import chalk = require('chalk');
+
+export default class Scan extends Command {
+  private _isVerbose = false;
```

Comment:
nit: if we're adopting an underscore prefix for private class members, let's be consistent within the codebase. We can add a new task to make sure everything is the same in a followup PR